app.amsterdam.json

```diff
@@ -93,6 +93,10 @@
       "maxZoom": 16,
       "minZoom": 7,
       "zoom": 7
+    },
+     "optionsAreaMap": {
+      "focusRadiusMeters": 100,
+      "zoom": 13
```

Comment:
Waarom geen maxZoom?

Reply:
Max zoom stays the same. So it takes the default of app.json